spec/requests/submissions_spec.rb

```diff
@@ -53,6 +53,19 @@
         get submissions_phase_path(phase)
         expect(response).to have_http_status(:not_found)
       end
+
+      it "renders submission statistics" do
+        submission = create(:submission, challenge: challenge, phase: phase)
+        submission_2 = create(:submission, challenge: challenge, phase: phase, judging_status: "selected")
```

Comment:
[Lint/UselessAssignment](https://docs.rubocop.org/rubocop/cops_lint.html#lintuselessassignment)
```suggestion
        create(:submission, challenge: challenge, phase: phase)
        create(:submission, challenge: challenge, phase: phase, judging_status: "selected")
```